domain-models-runtime/src/test/java/org/folio/rest/persist/MetadataTriggerIT.java

```diff
@@ -0,0 +1,183 @@
+package org.folio.rest.persist;
+
+import io.vertx.core.DeploymentOptions;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.sql.UpdateResult;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.client.TenantClient;
+import org.folio.rest.tools.utils.NetworkUtils;
+import org.folio.rest.tools.utils.VertxUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+
+import static io.vertx.core.Future.failedFuture;
+import static io.vertx.core.Future.succeededFuture;
+
+@RunWith(VertxUnitRunner.class)
+public class MetadataTriggerIT {
+
+  private static final String TENANT_ID = "test_tenant";
+  private static final String TABLE = "test_md_trigger_table";
+
+  private static Vertx vertx;
+
+  @BeforeClass
+  public static void setUp(TestContext context) {
+    vertx = VertxUtils.getVertxWithExceptionHandler();
+    try {
+      PostgresClient.getInstance(vertx).startEmbeddedPostgres();
```

Comment:
There is no need to unconditionally start embedded postgres.
Postgres is started in the pre-integration-test phase: https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/pom.xml#L252
This is an example how to setup a *IT.java test where embedded postgres is started only when needed, for example when running a single *IT.java only: https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientTransactionsIT.java#L33

Reply:
done